### ISSUE TITLE
 torchtext.datasets.IMDB.splits

### ISSUE THREAD

**Issue by Tikquuss** [pytorch/text#790]
By default, the following code split the IMDB dataset with ratio of 0.5 ( train_data =25000 and test_data=25000).  

### train_data, test_data = torchtext.datasets.IMDB.splits(TEXT, LABEL)  

I want to split it with ratio of .0.8 ( train_data =40000 and test_data=10000). How can I process?

**Comment by zhangguanheng66** [comment]
IMO, you don't want to change test dataset because it's used for benchmark. You might want to split train dataset.

**Comment by Tikquuss** [comment]
@zhangguanheng66  Thank you

**Comment by Esperanto-mega** [comment]
通过datasets.IMDB获取到的数据集应该是服从官网中给出的组织形式的，如果需要修改这个比例，需要下载数据集后进行修改。

![image](https://user-images.githubusercontent.com/57856207/128975364-ad3a303d-1a52-48d7-a14b-3f97b5792970.png)

**Comment by Tikquuss** [comment]
我宁愿让这个比例保持不变，并从训练数据集中抽取5000个例子来验证模型。有了这一点，该模型就能实现泛化。
